README.md

```diff
@@ -3,7 +3,6 @@
 ### Available Translations 🗣
 
 * [Pt-BR](https://bit.ly/3LI8kAc)
-* [zh-CN](translations/README.zh-CN.md)
```

Comment:
Wonder why you deleted the link from here. We will still need it for folks to access the translated version.

Reply:
Hmm......There are no important differences between "zh-CN" and "中文" in my opinion.
They are both Translated into Simplified Chinese  spoken in the Mainland of China and it can be called as "简体中文".
What we need is Traditional Chinese that our called "繁体中文",I think it's a better way than two zh-CNs or 中文s in similar form.

Reply:
In other way to say.These two translation works can be merged as one.